### PR TITLE
More dependencies for createrepo_c

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Required for #12003 
Blocked by #12004

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/drpm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    # https://abi-laboratory.pro/index.php?view=timeline&l=rpm
+    - {{ pin_compatible('rpm-tools', max_pin="x.x") }}
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libdrpm$SHLIB_EXT
+
+about:
+  home: https://github.com/rpm-software-management/drpm/
+  license: BDS-3-Clause
```
Comment (saraedum) [inline recipes/drpm/meta.yaml:43]: Isn't LGPLv2(+?) as mentioned in `COPYING` the main license?

**Comment by saraedum** [comment]
Is this something that should be fixed?
```
WARNING (drpm): run-exports library package conda-forge::cmocka-1.1.5-h516909a_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

File: recipes/libmodulemd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - file
+    - glib
+    - yaml
+    - rpm-tools
+    - gobject-introspection
+    - pygobject
+    - meson
+    - help2man
+  run:
+    - python
```
Comment (saraedum) [inline recipes/libmodulemd/meta.yaml:31]: Does the way you are using Python here actually work? I mean, this is compiled C but the Python bits are independent of that. Will conda understand how to install this in a different version of Python?
I was under the impression that one should always put python as a host & run dependency…but maybe that's the point here of putting it only into run?

File: recipes/libmodulemd/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    # https://abi-laboratory.pro/index.php?view=timeline&l=rpm
+    - {{ pin_compatible('rpm-tools', max_pin="x.x") }}
+    - {{ pin_compatible('file') }}
+
+test:
+  commands:
+    - modulemd-validator --help
+  imports:
+    - gi
```
Comment (saraedum) [inline recipes/libmodulemd/meta.yaml:41]: Doesn't this test that `gobject-introspection` has made it to `run` only? (Maybe that's intentional.)

File: recipes/libmodulemd/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+source:
+  url: https://github.com/fedora-modularity/libmodulemd/releases/download/libmodulemd-{{ version }}/modulemd-{{ version }}.tar.xz
+  sha256: cb86b1dad4f1578895225ba4ee435dbb7d75262898f69a08507b01759bfc81ab
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  ignore_run_exports:
+    - cmocka
```
Comment (saraedum) [inline recipes/libmodulemd/meta.yaml:16]: I think you also want this for the other recipe.

**Pull request merged**